interfaces/apparmor/backend.go

```diff
@@ -300,12 +300,12 @@ func profileGlobs(snapName string) []string {
 // - snap.core.NNNN.usr.lib.snapd.snap-confine
 // - var.lib.snapd.snap.core.NNNN.usr.lib.snapd.snap-confine
 // - snap-confine.core.NNNN
```

Comment:
it would be good though to mark somehow in this list which forms are current and which forms are historical (I understand we need to deal with all)